extension-readme.md

```diff
@@ -1,134 +1,97 @@
--   List and delete AWS CloudFormation stacks
--   List, invoke, and delete [AWS Lambda](https://aws.amazon.com/lambda/) functions
+![Add Debug Configuration Template](./resources/marketplace/Codelens-YAML-template.png)
 
-The following example shows a highlight of this functionality.
+###### The _CodeLens_ indicator in the SAM template allows you to add a debug configuration for the serverless application.</h6>
 
-After you have [created a serverless application](https://docs.aws.amazon.com/console/toolkit-for-vscode/create-sam), you can locally run, debug, and further develop that application in the VS Code editor. For example:
+Alternatively, you can run and debug just the AWS Lambda function and exclude other resources defined by the SAM template. Again, use a _CodeLens_ indicator for an AWS Lambda-function handler. (A _handler_ is a function that Lambda calls to start execution of a Lambda function.)
 
-![Configure and Run 1](./resources/marketplace/sam-configure-and-run-still-1.png)
+![Add Debug Configuration Direct](./resources/marketplace/Codelens-direct-function.png)
 
-Take special note of the _CodeLenses_ that enable you to run and debug the application locally.
+###### The _CodeLens_ indicator in the application file lets you add a debug configuration for a selected AWS Lambda function.
 
-The status and results are shown in the **OUTPUT** panel when the **AWS Toolkit** output channel is selected.
+When you run a debug session, the status and results are shown in the **OUTPUT** panel when the **AWS Toolkit** output channel is selected.
 
 ![Configure and Run 1](./resources/marketplace/sam-configure-and-run-still-2.png)
 
-When you're satisfied, you can [deploy your application](https://docs.aws.amazon.com/console/toolkit-for-vscode/deploy-serverless-app) to a CloudFormation stack and then [run the Lambda function](https://docs.aws.amazon.com/console/toolkit-for-vscode/remote-lambda) on AWS.
+###### After a local run is complete, the output appears in the **OUTPUT** tab.
+
+When you're satisfied with performance, you can [deploy your serverless application](https://docs.aws.amazon.com/console/toolkit-for-vscode/deploy-serverless-app). The SAM template is converted to a CloudFormation template, which is then used to deploy all the application's assets to the AWS Cloud.
 
-For full details, see the [AWS Explorer](https://docs.aws.amazon.com/toolkit-for-vscode/latest/userguide/aws-explorer.html) in the Toolkit's user guide.
+### Supported runtimes
 
-#### Additional Information about CodeLenses
+The Toolkit _local SAM debugging_ feature supports these runtimes:
 
-The functions that have Codelenses are those that use AWS Lambda-function handler syntax. A _handler_ is a function that Lambda calls to start execution of a Lambda function. These CodeLenses enable you to locally run or debug the corresponding serverless application. CodeLens actions in the Toolkit include:
+-   C# (.NET Core 2.1)
```

Comment:
This seems to be languages not runtimes (well runtimes are in the parenthesis)

Reply:
I replaced the "languages and runtimes" with just "runtimes" because "runtime" is the concept that is generally used through the extension.  For purposes of this doc, the distinction is just noise that readers don't care about.